src/main/java/com/xjeffrose/xio/client/HttpClientConnector.java

```diff
@@ -101,15 +98,11 @@ protected void initChannel(SocketChannel channel) throws Exception {
         cp.addLast("httpClientCodec", new HttpClientCodec());
 //        cp.addLast("defaltor", new HttpContentDecompressor());
         cp.addLast("chunkAggregator", new HttpObjectAggregator(maxFrameSize));
-        cp.addLast("idleTimeoutHandler", new IdleStateHandler(
-            20000,
-            NO_WRITER_IDLE_TIMEOUT,
-            NO_ALL_IDLE_TIMEOUT,
-            TimeUnit.MILLISECONDS));
-        cp.addLast("idleDisconnectHandler", new IdleDisconnectHandler(
-            2000,
-            NO_WRITER_IDLE_TIMEOUT,
-            NO_ALL_IDLE_TIMEOUT));
+        cp.addLast("idleDisconnectHandler", new XioIdleDisconnectHandler(
+              20000,
```

Comment:
I'm actually not sure if this is the right timeout.  2,000 seconds seemed like a long time.